Gemfile

```diff
@@ -3,7 +3,7 @@ source 'https://rubygems.org'
 ruby RUBY_VERSION # Needed to consider & solve for Ruby version requirements
 
 gem 'berkshelf'
-gem 'chef', '~> 12.0'
+gem 'chef', '>= 12.0'
```

Comment:
>= **12**?

Reply:
Because now 12 is the minimum requirement.

Chef 13 still requires ruby 2.4.1 that we do not want to enforce right now.